src/MergeContext.js

```diff
@@ -787,9 +787,24 @@ class PullRequest {
         return (this._rawPr.title + ' (#' + this._rawPr.number + ')' + '\n\n' + this._prBody()).trim();
     }
 
+    // Searches for the first non_ASCII_printable character and returns
+    // its position in the string. By default, JavaScript strings are
```

Comment:
Please remove the "By default..." sentence. The internal String "representation" (it is actually more like "interpretation" -- String storage is probably always an array of integers), especially the "default" one, seems irrelevant for this function interface.